### ISSUE TITLE
Create Anaconda3 (Python 3.5) tool dependency package

### ISSUE THREAD

**Issue by BrunoGrandePhD** [morinlab/galaxy_tools#18]
Working off of https://testtoolshed.g2.bx.psu.edu/view/iuc/package_anaconda_2_3_0/d3f29b11da06

**Comment by BrunoGrandePhD** [comment]
I first tried creating a package for Anaconda3 4.1.1, but ran into the following error:

`Python 3.5.2 :: Continuum Analytics, Inc.
ERROR: placeholder '/home/ilan/minonda/envs/_build_placehold_placehold_placehold_placehold_placehold' too short in: curl-7.49.0-0`

I found this [GitHub issue](https://github.com/ContinuumIO/anaconda-issues/issues/927) that explains what happened. Basically, the install directory on our Nova Galaxy instance is longer than the 80 characters allowed by the new prefix. 

I'm now trying again using Anaconda 4.0.0, which uses an older version of the curl package that doesn't have this prefix.
